### ISSUE TITLE
How to draw a networkx graph which has user defined nodes?

### ISSUE THREAD

**Issue by callmexss** [pyviz/hvplot#216]
Hello there,

I have a user defined Node class which has some specific attributions to simulate a network. And I try to use the hvnx.draw(G) to draw a picture.  And it gives me exceptions as follows:

```sh
TypeError                                 Traceback (most recent call last)
<ipython-input-54-52c9eff9e40e> in <module>
      1 plt.figure(figsize=(8, 8))
      2 # nx.draw_networkx(G, pos=pos, labels=dict(zip(nodes, [str(node.loc)[:3] for node in nodes])))
----> 3 hvnx.draw_networkx(G, pos)
      4 plt.title("typology for 50 nodes".title())
      5 plt.savefig("./pictures/typology-for-50-nodes.png", dpi=90)

~/.local/share/virtualenvs/SU4szOQM/lib/python3.6/site-packages/hvplot/networkx.py in draw_networkx(G, pos, **kwargs)
    369     """
    370     kwargs['_axis_defaults'] = False
--> 371     return draw(G, pos, **kwargs)
    372 
    373 

~/.local/share/virtualenvs/SU4szOQM/lib/python3.6/site-packages/hvplot/networkx.py in draw(G, pos, **kwargs)
    223 
    224     # Construct Graph object
--> 225     g = _from_networkx(G, pos, **params)
    226 
    227     if 'nodelist' in kwargs:

~/.local/share/virtualenvs/SU4szOQM/lib/python3.6/site-packages/hvplot/networkx.py in _from_networkx(G, positions, nodes, cls, **kwargs)
     76 
     77     # Unpack node positions
---> 78     for idx, pos in sorted(positions.items()):
     79         node = G.nodes.get(idx)
     80         if node is None:

TypeError: '<' not supported between instances of 'Node' and 'Node'
```
And i went to the source code change
```py
 for idx, pos in sorted(positions.items()):
```
to 
```py
 for idx, pos in sorted(positions.items(), key=lambda x: x[0].loc):
```
and
```py
 for idx, pos in positions.items():
```
and I also define a magic function `__lt__` in class Node, which all failed.
So how can I draw a Graph which connects with user defined nodes?

**Comment by philippjfr** [comment]
I'm not sure it should even be sorting there, will have to look.

**Comment by jsignell** [comment]
I imagine this was fixed as a consequence of #284. If it isn't fixed, please reopen and provide a reproducible example.
